docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+| :----------------------------------- |:------:| -----------------------------------------------------------------:|:------------------------------------------|
+| 'tissue'                             | string | UBERON term                                                       | `area postrema`                           |
+| 'tissue_ontology_term_id'            | string | UBERON term id                                                    | `UBERON:0002162`                          |
+| 'assay'                              | string | EFO term                                                          | `scRNA-seq`                               |   
+| 'assay_ontology_term_id'             | string | EFO term id                                                       | `EFO:0008913`                             |
+| 'disease'                            | string | MONDO term or `normal`                                            | `kuru`                                    |
+| 'disease_ontology_term_id'           | string | MONDO term id or `PATO:0000461`                                   | `MONDO:0006825`                           |
+| 'cell_type'                          | string | CL term (or UBERON term when no CL term is available)             | `excitatory neuron`                       |
+| 'cell_type_ontology_term_id'         | string | CL term id (or UBERON term id when no CL term id is is available) | `CL:0008030`                              |
+| 'sex'                                | string | `male`, `female`, `mixed`, `unknown`, or `other`                  | `mixed`                                   |
+| 'ethnicity'                          | string | HANCESTRO term, `na` if non-human, `unknown` if not available     | `genetically isolated population`         |
+| 'ethnicity_ontology_term_id	'        | string | HANCESTRO term id, `na` if non-human                              | `HANCESTRO:0290`                          |
+| 'development_stage'                  | string | HsapDv term, `unknown` if not available                           | `9th week post-fertilization human stage` |
+| 'development_stage_ontology_term_id	'| string | HsapDv term id if human, child of `EFO:0000399` otherwise         | `HsapDv:0000046`                          |
+
+<div align="center">
+  <b> Table: </b> Required cell level metadata
+</div>
+
+<br/>
+
+In the above table, the types of cell level metadata that is collected for the schema. In particular, it is imporant to exapnd on how to annotate `cell_type` field. In general, we try to find most specific ontology term that accurately represents a given cell type in your dataset. In the event that your cell type is not described accurately by any term in the `CL` ontology, then it is sufficient to report the most accurate `UBERON` term and id instead. In some cases, when neither the `CL` nor the `UBERON` term are descriptive enough, then it is possible to submit an empty string (`''`) for that entry.
```

Comment:
_imporant to exapnd -> important to expand_

_find most -> find the most_

Reply:
_In the event that your cell type is not described accurately by any term in the `CL` ontology, then it is sufficient to report the most accurate `UBERON` term and id instead._

Where is that documented in the schema?

Reply:
the typos have been fixed

@jahilton I amended the wording to be more precise about where to include the empty string. Let me know if that doesn't solve the clarity issue

@jahilton the scenarios where I have had two scenarios where researchers specifically ask me to use an `UBERON` term instead of a `CL` term. The first is when a common cell type (i.e. `myofibroblasts`) is detected in a very specific tissue. The `CL` term in this case was insufficient and the researchers opted to annotate the cell type according to the tissue they believed the cell to be derived from. The second scenario was where the researchers believed that they had detected a novel cell type.

I don't believe it is documented in the schema, but we have had internal discussions on the matter. Additionally, the CZI curation tools will both apply and validate an `UBERON` term/id in the cell type field, so I think acceptance in the schema is implied here (I am not however saying that I am favoring its inclusion in the schema).

Reply:
From curation sync:

empty strings - Never!

stick with cl and co-annotate with the tissue term

max to follow up on fawkner-corbett use-case